damnit/gui/main_window.py

```diff
@@ -893,6 +896,22 @@ def export_selection_to_zulip(self):
         df.replace(["None", '<NA>', 'nan'], '', inplace=True)
         self.zulip_messenger.send_table(df)
 
+    adeqt_window = None
+
+    def show_adeqt(self):
+        try:
+            from adeqt import AdeqtWindow
```

Comment:
This feels like we're just making things harder for ourselves :thinking: I'd add it as a dependency.

Reply:
Yeah, if we're making it visible in a menu, that definitely makes sense. :+1: